libs/wire-api/src/Wire/API/Error/Galley.hs

```diff
@@ -0,0 +1,228 @@
+
+type instance MapError 'CustomBackendNotFound = 'StaticError 404 "custom-backend-not-found" "Custom backend not found"
+
+type instance MapError 'DeleteQueueFull = 'StaticError 503 "queue-full" "The delete queue is full; no further delete requests can be processed at the moment"
+
+type instance MapError 'TeamSearchVisibilityNotEnabled = 'StaticError 403 "team-search-visibility-not-enabled" "Custom search is not available for this team"
+
+type instance MapError 'CannotEnableLegalHoldServiceLargeTeam = 'StaticError 403 "too-large-team-for-legalhold" "Cannot enable legalhold on large teams (reason: for removing LH from team, we need to iterate over all members, which is only supported for teams with less than 2k members)"
+
+-- We need this to document possible (operation denied) errors in the servant routes.
+type family MissingPermissionMessage (a :: Maybe Perm) :: Symbol where
+  MissingPermissionMessage ('Just p) = "Insufficient permissions (missing " `AppendSymbol` Show_ p `AppendSymbol` ")"
+  MissingPermissionMessage 'Nothing = "Insufficient permissions"
+
+--------------------------------------------------------------------------------
+-- Authentication errors
+
+data AuthenticationError
+  = ReAuthFailed
+  | VerificationCodeAuthFailed
+  | VerificationCodeRequired
+
+type instance MapError 'ReAuthFailed = 'StaticError 403 "access-denied" "This operation requires reauthentication"
+
+type instance MapError 'VerificationCodeAuthFailed = 'StaticError 403 "code-authentication-failed" "Code authentication failed"
+
+type instance MapError 'VerificationCodeRequired = 'StaticError 403 "code-authentication-required" "Verification code required"
+
+instance IsSwaggerError AuthenticationError where
+  addToSwagger =
+    addStaticErrorToSwagger @(MapError 'ReAuthFailed)
+      . addStaticErrorToSwagger @(MapError 'VerificationCodeAuthFailed)
+      . addStaticErrorToSwagger @(MapError 'VerificationCodeRequired)
+
+type instance ErrorEffect AuthenticationError = Error AuthenticationError
+
+authenticationErrorToDyn :: AuthenticationError -> DynError
+authenticationErrorToDyn ReAuthFailed = dynError @(MapError 'ReAuthFailed)
+authenticationErrorToDyn VerificationCodeAuthFailed = dynError @(MapError 'VerificationCodeAuthFailed)
+authenticationErrorToDyn VerificationCodeRequired = dynError @(MapError 'VerificationCodeRequired)
+
+instance Member (Error DynError) r => ServerEffect (Error AuthenticationError) r where
+  interpretServerEffect = mapError authenticationErrorToDyn
+
+--------------------------------------------------------------------------------
+-- Team feature errors
+
+data TeamFeatureError
+  = AppLockInactivityTimeoutTooLow
+  | LegalHoldFeatureFlagNotEnabled
+  | LegalHoldWhitelistedOnly
+  | DisableSsoNotImplemented
+  | FeatureLocked
+
+instance IsSwaggerError TeamFeatureError where
+  -- Do not display in Swagger
```

Comment:
I guess there's a good reason for this, but can you please clarify?

Reply:
It's just the way it used to be. I guess those errors are to be considered somewhat internal?